physionet-django/user/models.py

```diff
@@ -473,6 +473,24 @@ def get_users_with_permission(app_label, permission_codename):
 
         return users
 
+    def get_credentialing_status(self):
+        """
+        Returns the credentialing status of the user.
+        If the user is not credentialed, returns the status of the user's credentialing application.
+        """
+        if self.is_credentialed:
+            return 'Credentialed'
+        else:
+            application = self.credential_applications.last()
+            if not application:
+                return 'No application found'
+            application_status = application.status
```

Comment:
Sorry Amit, I don't think this line is necessary (and for me it makes the code less readable). 

Before I merge, please could you drop the `application_status` variable and instead use `application.status`? 

(unless I'm missing a good reason for assigning `application_status`!

Reply:
Ah yes, i agree, earlier i think there were multiple if else statement so i had stored this in a variable.

Let me change and rebase quickly